import.py

```diff
@@ -644,8 +655,22 @@ def compile_base(compile_script: str, source: str, c_file: str, out_file: str) -
         print("Warning: failed to compile .c file.")
 
 
+def write_custom_weight_comments(compiler_type: str, filename: str) -> None:
+    if os.path.exists("default_weights.toml"):
+        with open("default_weights.toml") as f:
+            all_weights = toml.load(f)
+            compiler_weights = all_weights[compiler_type]
+
+        with open(filename, "a", encoding="utf-8") as f:
+            f.write("# uncomment lines below to customize the weights\n")
+            f.write("# run --help=randomization to see doc\n")
+            f.write("[custom_weights]\n")
+            for randomization_type, weight in compiler_weights.items():
+                f.write("# " + randomization_type + " = " + str(weight) + "\n")
+
+
 def write_to_file(cont: str, filename: str) -> None:
-    with open(filename, "w", encoding="utf-8") as f:
+    with open(filename, "a", encoding="utf-8") as f:
```

Comment:
ehh, let's do string concatenation in the caller instead

Reply:
done